formant_ros2_adapter/scripts/components/subscriber/ingester.py

```diff
@@ -63,7 +65,8 @@ def ingest(
         msg_timestamp: int,
         tags: Dict,
     ):
-
+        self.ingestion_analytics.log_received_message(topic)
+        self.ingestion_analytics.log_sent_message(formant_stream)
```

Comment:
Let's only log a sent message directly **after** the `fclient._post_<type>` calls to ensure its actually sent